.github/workflows/push-image-frontend.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - 'develop'
       - 'master'
+    tags:
```

Comment:
@pinarruiz can you please use tags-ignore here instead of a wildcard ?

Reply:
The asterisk is a special character in YAML. It is used in conjunction with &, which defines an anchor, that can then be referenced it with an asterisk (*) .  Therefore we can enclose it in single quotes.  Also I was thinking that every commit to develop need not get built. So we can pass a tag-ignore that ignores the workflow build on a specific tag. like "build : ignore" or something around that lines

Reply:
I dont think i fully understand, should not that be accomplished by adding a certain string to the commit message? Like: 

- `[skip ci]`
- `[ci skip]`
- `[no ci]`
- `[skip actions]`
- `[actions skip]`

As stated by: [skipping-workflow-runs](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs)